enterprise/backend/test/metabase_enterprise/sso/api/saml_test.clj

```diff
@@ -0,0 +1,19 @@
+(ns metabase-enterprise.sso.api.saml-test
+  (:require
+   [clojure.test :refer :all]
+   [metabase-enterprise.sso.integrations.saml-test :refer [client-full-response
+                                                           with-saml-default-setup]]))
+
+(deftest saml-settings-test
+  (testing "PUT /auth/saml/settings"
+    (testing "Valid SAML settings can be saved via an API call"
+      (with-saml-default-setup
+        (client-full-response :put 200 "/auth/saml/settings" {:saml-keystore-path "test_resources/keystore.jks"
```

Comment:
Not sure how to include a superuser here, attempting to add `:crowberto` user credentials leads to

```
; ERROR in metabase-enterprise.sso.api.saml-test/saml-settings-test (http_client.clj:17):
; Uncaught exception, not in assertion
; error: clojure.lang.ExceptionInfo: Failed to authenticate with credentials {:credentials {:username "crowberto@metabase.com", :password "blackjet"}} (http_client.clj:17)
```

Reply:
I think here you should just use `metabase.test/user-http-request` which is the standard way of doing requests in tests and is ultimately a wrapper around `metabase.http-client/client`. Not sure why the existing SAML tests define their own wrappers, probably just for convenience/historical reasons.

Reply:
I need to get rid of the /api from the URL prefix, like client-full-response from saml-test.